packages/account/src/Sections/Security/LoginHistory/login-history.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Loading, ThemedScrollbars } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { WS } from '@deriv/shared';
+import LoadErrorMessage from 'Components/load-error-message';
+import LoginHistoryContent from './login-history-content';
+import { getLoginHistoryFormattedData } from '../../../../../utils/src/getLoginHistoryFormattedData';
```

Comment:
Why is relative path being used?

Reply:
i checked with '@deriv/utils'. build is failing 🏗️